option.go

```diff
@@ -282,6 +302,61 @@ func (option *Option) Set(value *string) error {
 	return convert("", option.value, option.tag)
 }
 
+// Set the value of a "terminated" option to the specified value.
+// An error will be returned if the specified value could not be
+// converted to the corresponding option value type.
+// If option value is slice of slice, the parameter is converted
+// as a slice and appended to the option value. Otherwise, if
+// option value is slice, option value is replaced by parameter.
+func (option *Option) SetTerminatedOption(value []*string) error {
+	tp := option.value.Type()
+
+	if tp.Kind() != reflect.Slice {
+		return newErrorf(ErrInvalidTag,
+			"terminated flag `%s' must be a slice or slice of slices",
+			option.shortAndLongName())
+	}
+
+	if len(value) == 0 {
+		return newErrorf(ErrExpectedArgument,
+			"expected argument for flag `%s'",
+			option.shortAndLongName())
+	}
+
+	if option.clearReferenceBeforeSet {
+		option.empty()
+	}
+
+	option.isSet = true
+	option.preventDefault = true
+	option.clearReferenceBeforeSet = false
+
+	elemtp := tp.Elem()
+
+	if elemtp.Kind() == reflect.Slice {
+		elemvalptr := reflect.New(elemtp)
+		elemval := reflect.Indirect(elemvalptr)
```

Comment:
`elemvalptr` is only used to create `elemval`, so why not just merge into one line? Also, probably `elemval` -> `elemVal` to fit Go naming style.

```suggestion
		elemVal := reflect.Indirect(reflect.New(elemTp))
```